src/app/admin/admin-reports/filtered-items/filtered-items.component.ts

```diff
@@ -0,0 +1,301 @@
+import { Component, ViewChild } from '@angular/core';
+import { FormArray, FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { NgbAccordion } from '@ng-bootstrap/ng-bootstrap';
+import { TranslateService } from '@ngx-translate/core';
+import { map, Observable } from 'rxjs';
+import { CollectionDataService } from 'src/app/core/data/collection-data.service';
+import { CommunityDataService } from 'src/app/core/data/community-data.service';
+import { MetadataFieldDataService } from 'src/app/core/data/metadata-field-data.service';
+import { MetadataSchemaDataService } from 'src/app/core/data/metadata-schema-data.service';
+import { RestRequestMethod } from 'src/app/core/data/rest-request-method';
+import { DspaceRestService } from 'src/app/core/dspace-rest/dspace-rest.service';
+import { RawRestResponse } from 'src/app/core/dspace-rest/raw-rest-response.model';
+import { MetadataField } from 'src/app/core/metadata/metadata-field.model';
+import { MetadataSchema } from 'src/app/core/metadata/metadata-schema.model';
+import { Collection } from 'src/app/core/shared/collection.model';
+import { Community } from 'src/app/core/shared/community.model';
+import { Item } from 'src/app/core/shared/item.model';
+import { getFirstSucceededRemoteListPayload } from 'src/app/core/shared/operators';
+import { isEmpty } from 'src/app/shared/empty.util';
+import { environment } from 'src/environments/environment';
+import { FiltersComponent } from '../filters-section/filters-section.component';
+import { FilteredItems } from './filtered-items-model';
+import { OptionVO } from './option-vo.model';
+import { PresetQuery } from './preset-query.model';
+import { QueryPredicate } from './query-predicate.model';
+
+@Component({
+  selector: 'ds-report-filtered-items',
+  templateUrl: './filtered-items.component.html',
+  styleUrls: ['./filtered-items.component.scss']
+})
+export class FilteredItemsComponent {
+
+  collections: OptionVO[];
+  presetQueries: PresetQuery[];
+  metadataFields: OptionVO[];
+  metadataFieldsWithAny: OptionVO[];
+  predicates: OptionVO[];
+  pageLimits: OptionVO[];
+
+  queryForm: FormGroup;
+  currentPage: number = 0;
```

Comment:
Because this is a simple integer, you can remove the type here. It should just be `currentPage = 0;`  Again, this error can be seen via `yarn lint`